metrics/gender_data_metrics.tsv

```diff
@@ -0,0 +1,273 @@
+month	female total	male total	gender_diverse total	female_MoM_difference	gender_diverse_MoM_difference	male_MoM_difference	female + gender_diverse	all_genders_sum	% of new articles about gender minorities
```

Comment:
In naming columns, I'm inclined to favor clarity over brevity, and make sure that the column names are clear even when separated from the name of the file (as when you use `load_all_metric_files`, for example).

In this case, I'd suggest (using underscores since that's the norm in the existing files):
* female total -> quality_articles_about_women
* male total -> quality_articles_about_men
* ...
* net_new_quality_articles_about_men
* ...
* gender_minorities_share_of_net_new_quality_gendered_articles (this one is a mouthful, I admit, but I feel like it's better than using an imprecise name that might confuse people. Definitely open for debate.)

And is it important to keep all the intermediate columns (everything other than the first three and the last one)? I suppose it doesn't cost anything, but maybe there are a few that aren't really useful.